e2e-tests/support/tasks.js

```diff
@@ -10,8 +10,7 @@ module.exports = {
 
     return {
       log(message) {
-        // eslint-disable-next-line no-console
-        console.log(message);
+        console.info('Cypress create tasks %s', message);
```

Comment:
I'm not sure I follow what `Cypress create tasks` is referring to in this context. This `log` function isn't logging anything to do with creating tasks, but instead can be used within Cypress tests to log something to the test output. e.g.
```
cy.task('log', 'some debug message');
```

Reply:
Have reverted.